### PR TITLE
new landing page

### PR DESCRIPTION
**Pull request by Alfaj2302**

### CODE REVIEW

File: src/app/tenant/sunbird/index.html [hunk 1]
```diff
@@ -1,211 +1,388 @@
...
-                            <img style="width: 139px;" src="images/AKAAM_.png" alt="Logo Center" class="img-fluid"/>
-                        </a>
+    <section>
+        <div class="">
+            <nav class="navbar">
+                <div class="p-0 col-12 col-md-5 col-lg-4 ps-2">
+                    <div class="d-flex gap-2 col-12 align-items-center justify-content-evenly">
+                        <div class="nav_img">
+                            <a href="#">
+                                <img src="images/national_logo.png" width="199px" alt="Logo" />
```
Comment (Ekta29yadav) [inline src/app/tenant/sunbird/index.html:55]: don't give hard coded width

File: src/app/tenant/sunbird/index.html [hunk 2]
```diff
@@ -1,211 +1,388 @@
...
+                            </a>
+                        </div>
+                        <div class="nav_img">
+                            <a href="https://mohua.gov.in" target="_blank">
+                                <img style="width:140px;" src="images/MoHUA_.png" alt="Logo Center" class="" />
```
Comment (Ekta29yadav) [inline src/app/tenant/sunbird/index.html:60]: don't give hard coded width

File: src/app/tenant/sunbird/index.html [hunk 3]
```diff
@@ -1,211 +1,388 @@
...
+                        </div>
                     </div>
                 </div>
-                <div class="sixteen wide tablet six wide computer column sb-right-menu justify-content-center">
-                    <div class="flex-center">
-                        <a target="_blank" href="/learnathon.html" class="learn-padding text-white">Visit Learnathon</a>
+                <div class="pe-4 ps-2 d-none navItems d-lg-flex justify-content-between align-items-center col-8 ">
+                    <div class="input-group search mb-3">
+                        <span class="input-group-text border-0  bg-transparent" id="basic-addon1">
+                            <img src="NULP Landing page assets/Icons/Icon_Search.svg" class="bg-color-none" width="30px"
```
Comment (Ekta29yadav) [inline src/app/tenant/sunbird/index.html:68]: don't give hard coded width

Comment (Ekta29yadav) [reply src/app/tenant/sunbird/index.html:68]: please check src path

File: src/app/tenant/sunbird/index.html [hunk 4]
```diff
@@ -1,211 +1,388 @@
...
-                            <h1 class="sb-text-upper mt-0">Learning Platform</h1>
-                            <p class="sb-font">Launching on 13th March 2023 where cities and states share solutions <br> to foster collaborating learning</p>
-                            <div><a class="sb-btn-default mt-10" href="/resources?learnathon">Sign In</a>
-                            <a class="sb-btn-default mt-10 ml-20" target="_blank" href="./learnathon.html">Learnathon</a>
+
+                <div class="menu d-flex col-12 col-md-6 justify-content-evenly d-lg-none">
+
+                    <div class="input-group search mb-3">
+                        <span class="input-group-text border-0  bg-transparent" id="basic-addon1">
+                            <img src="NULP Landing page assets/Icons/Icon_Search.svg" class="bg-color-none" width="30px"
```
Comment (Ekta29yadav) [inline src/app/tenant/sunbird/index.html:85]: don't give hard coded width and also check src path

File: src/app/tenant/sunbird/index.html [hunk 5]
```diff
@@ -1,211 +1,388 @@
...
+            </div>
+        </div>
+    </section>
+
+    <section>
+        <div class="nulp-work">
+            <div class="container">
+                <div class="main row">
+                    <div class="work-img left-wroking row col-12 col-lg-6">
+                        <img class="h-100" src="NewLandingAssets/How NULP works (1).jpg" alt="">
```
Comment (Ekta29yadav) [inline src/app/tenant/sunbird/index.html:165]: also check src path

File: src/app/tenant/sunbird/index.html [hunk 6]
```diff
@@ -225,277 +402,342 @@ <h4 class="sb-se-title mt-55 py-32 pb-10 pl-32 mb-20">Our Partners</h4>
...
+            <div ngxSlickItem class="slide">
+                <div class="item">
+                    <div style="justify-content: center" class="d-flex">
+                        <div class="ui card p-0">
+                            <div class="bg-img-cover">
+                                <img src="images/abstract.svg" class="card-bg" />
+                            </div>
+                            <div class="small-bow">
+                                <a class="slider_info">
+                                    <img class="img-fluid card-img-top" src="im" width="305px"
```
Comment (Ekta29yadav) [inline src/app/tenant/sunbird/index.html:421]: don't give hard coded width

File: src/app/tenant/sunbird/index.html [hunk 7]
```diff
@@ -225,277 +402,342 @@ <h4 class="sb-se-title mt-55 py-32 pb-10 pl-32 mb-20">Our Partners</h4>
...
+            <div ngxSlickItem class="slide">
+                <div class="item">
+                    <div style="justify-content: center" class="d-flex">
+                        <div class="ui card p-0">
+                            <div class="bg-img-cover">
+                                <img src="images/abstract.svg" class="card-bg" />
+                            </div>
+                            <div class="small-bow">
+                                <a class="slider_info">
+                                    <img class="img-fluid card-img-top" src="images/1.png" width="305px"
```
Comment (Ekta29yadav) [inline src/app/tenant/sunbird/index.html:446]: don't give hard coded width

File: src/app/tenant/sunbird/index.html [hunk 8]
```diff
@@ -225,277 +402,342 @@ <h4 class="sb-se-title mt-55 py-32 pb-10 pl-32 mb-20">Our Partners</h4>
...
+                                        style="height: 158px; object-fit: cover" />
```
Comment (Ekta29yadav) [inline src/app/tenant/sunbird/index.html:472]: don't give hard coded width

File: src/app/tenant/sunbird/index.html [hunk 9]
```diff
@@ -225,277 +402,342 @@ <h4 class="sb-se-title mt-55 py-32 pb-10 pl-32 mb-20">Our Partners</h4>
...
-                    <p class="fs-0-925 mt-15">Ministry of Housing and Urban Affairs, Government of India</p>
-                    <p class="fs-0-925">Nirman Bhawan, New Delhi- 110001, INDIA</p>
-                    <h4 class="fs-0-925 font-weight-bold pt-18">Anchor Institute:</h4>
+    <section>
+        <div class="container partners">
+            <div>
+                <h4 class="sb-se-title">Gallery</h4>
+            </div>
+            <div class="owl-carousel Gallery-carousel pt-3 pb-3 owl-theme">
+                <div class="item"><img class="" src="NewLandingAssets/Gallery (1).jpg" alt=""></div>
```
Comment (Ekta29yadav) [inline src/app/tenant/sunbird/index.html:546]: also check src path

File: src/app/tenant/sunbird/index.html [hunk 10]
```diff
@@ -225,277 +402,342 @@ <h4 class="sb-se-title mt-55 py-32 pb-10 pl-32 mb-20">Our Partners</h4>
...
-                        <a href="https://twitter.com/NIUA_India"  target="_blank" ><i class="icon twitter"></i></a>
+    <footer>
+        <div class="footer-bg">
+            <div class="container">
+                <div class="">
+                    <div class="row row-cols-md-2 justify-content-between">
+                        <div class="footer-anchors row">
+                            <a class="pb-2 text-dark col-12 col-sm-6 col-md-3"
+                                href="https://nulp.niua.org/aboutus.html">About Us</a>
+                            <a class="pb-2 text-dark col-12 col-sm-6 col-md-3" href="https://niua.in/">NIUA</a>
```
Comment (Ekta29yadav) [inline src/app/tenant/sunbird/index.html:580]: open the links in new window

File: src/app/tenant/sunbird/css/custom.css [hunk 11]
```diff
@@ -1168,7 +1168,7 @@ h1, h2, h3, h4, h5,
 }
 .footer a{
     color: #fff !important;
-    text-decoration: none;
+    /* text-decoration: none; */
```
Comment (komalm) [inline src/app/tenant/sunbird/css/custom.css:1171]: @Ekta29yadav These changes move into the tenant repo- https://github.com/NIUANULP/nulp-tenant

File: src/app/tenant/sunbird/newLandingPage.html [hunk 12]
```diff
@@ -0,0 +1,953 @@
+<!DOCTYPE html>
```
Comment (komalm) [inline src/app/tenant/sunbird/newLandingPage.html:1]: @Ekta29yadav - rename the original index.html file and add these changes to the index.html

**Pull request closed without merging**